definitions/salesforce.js

```diff
@@ -1,8 +1,11 @@
-
-    
 const salesforcePackage = require("../");
-  
-salesforcePackage({
-    inputDatasetName: 'salesforce',
-    inputDatasetSchema: 'salesforce_playground'
-})
\ No newline at end of file
+
+const salesforceModel = salesforcePackage({
+  salesforceSchema: "salesforce_playground",
```

Comment:
One thing I think could be more clear - there are two options here, the default output schema, and the schema of the tables generated by stitch/fivetran. We probably should by default output somewhere else.

One option here (and for when you do add e.g. fivetran support) would be to call this e.g. `stitchSalesforceSchema`.
Then when a user has fivetran data instead, they set `fivetranSalesforceSchema`.